README.md

```diff
@@ -87,16 +87,21 @@ is performed using the [pkg/errors][7] library.
 
 ## Acceptance Tests
 
-_**NOTE:** The tests are not currently well-structured, and depend on many
-hard-coded specifics of the Joyent Public Cloud (JPC) which is being shut down
-in November 2019.  It is likely not possible to run the test suite against a
-local Triton installation at this time, though that is definitely the intended
-test suite target environment for future development._
-
-Acceptance Tests run directly against the Triton API, so you will need either a
-local installation of Triton or an account with Joyent's Public Cloud offering
-in order to run them. The tests create real resources (and thus cost real
-money)!
+Acceptance Tests run directly against the Triton API (e.g. CloudAPI), so you
+will need an installation of Triton in order to run them, or
+[COAL](https://github.com/joyent/triton/blob/master/docs/developer-guide/coal-setup.md)
+(Cloud On A Laptop). The tests create real resources and thus could cost real
+money if your using a paid Triton account! It is also possible that the
```

Comment:
if you are

Reply:
Wow! Pedro picking up on my bad English grammer! I'm impressed!